### DISCUSSION TITLE
Bayes Factors as floats (in stead of strings)

### DISCUSSION THREAD

**Discussion by Andre3582** [raphaelvallat/pingouin#313] [Q&A] [answered]
Hi! 

I've noticed that Bayes Factors are put in the output df's as strings, rather than floats (like most other output statistics are). Is there a reason for this?

It seems more intuitive (and useful) if the Bayes Factors are a put in the df as numeric value. Is this something you would consider doing?

Thanks!
Andre

**Comment by raphaelvallat** [comment, marked as answer]
Hi @Andre3582,

This is done on purpose to avoid an annoying behavior where BF with very large or very small values force pandas to display the entire dataframe in scientific notation. However, you can disable this behavior with:

``python
pg.options['round.column.BF10'] = None
```

For more details on these options, please see [this notebook](https://github.com/raphaelvallat/pingouin/blob/b1c334d93f8f7f8b13c39199c08a7d8b619afd95/notebooks/06_Rounding.ipynb)
